### PR TITLE
[14.0] ADD ir attachement action download mixin

### PR DESCRIPTION
**Pull request by petrus-v**
Add a mixin to help developers to give a way to download multiple files at once from list view

### CODE REVIEW

**Comment by petrus-v** [comment]
I've rebase and an example how to rename path in archive !

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

File: attachment_zipped_download/tests/models/__inti__.py
Comment (pedrobaeza) [inline attachment_zipped_download/tests/models/__inti__.py:1]: Incorrect file name.

**Review by HekkiMelody — CHANGES_REQUESTED** [review]
I think something happened to your `__manifest__.py` and `__init__.py`  at the base of the module.

**Comment by petrus-v** [comment]
> I think something happened to your `__manifest__.py` and `__init__.py` at the base of the module.

ooops ! Indeed...

**Comment by petrus-v** [comment]
> > I think something happened to your `__manifest__.py` and `__init__.py` at the base of the module.
> 
> ooops ! Indeed...

@aleuffre hum... no in fact this PR do not change those files that why you don't see it in this PR I suppose, why are you telling that ? do you get unexpected behavior somehow ?

**Comment by HekkiMelody** [comment]
@petrus-v Sorry! I guess when I went to the "files changed" tab I remembered this being an entire new module for some reason. My bad.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Pull request merged**